tests/python/unittest/test_gluon_rnn.py

```diff
@@ -23,7 +23,7 @@
 import unittest
 from mxnet.test_utils import almost_equal, assert_almost_equal
 from common import assert_raises_cudnn_not_satisfied
-
+from tests.python.unittest.common import with_seed
```

Comment:
Can we merge this line into L25?
```python
from common import assert_raises_cudnn_not_satisfied, with_seed
```

Reply:
bad oversight on my behalf. Thank you for catching it!